store/sqlstore/channel_store.go

```diff
@@ -2425,48 +2425,64 @@ func (s SqlChannelStore) GetMembersForUserWithPagination(teamId, userId string,
 	return dbMembers.ToModel(), nil
 }
 
-func (s SqlChannelStore) AutocompleteInTeam(teamId string, term string, includeDeleted bool) (*model.ChannelList, error) {
-	deleteFilter := "AND Channels.DeleteAt = 0"
+func (s SqlChannelStore) AutocompleteInTeam(teamId string, userId string, term string, includeDeleted bool) (*model.ChannelList, error) {
```

Comment:
hi, in this function, I parse userId as parameter.
Is it the correct way @agnivade  ?

Reply:
because previous code didn;t handle private channel (only public channel)

Reply:
Yes, this is correct.